README.md

```diff
@@ -17,21 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] String
-[] Boolean
-[] Undefined
+[x] String
+[x] Boolean
+[x] Undefined
 [] NaN
-[] Number
-[] Array
-[] Null
+[x] Number
+[x] Array
+[x] Null
 ```
```

Comment:
:+1: Although remember an array isn't a "primitive" data type. But we didn't ask you to specify that here, did we!